pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -96,6 +96,10 @@ type BundleTarget struct {
 	// data will be synced to.
 	ConfigMap *KeySelector `json:"configMap,omitempty"`
 
+    // Secret is the target Secret in Namespaces that all Bundle source
+    // data will be synced to.
+    Secret *KeySelector `json:"secret,omitempty"`
+
```

Comment:
suggestion: We'll probably need to add validation to the [webhook](https://github.com/cert-manager/trust-manager/blob/main/pkg/webhook/validation.go) to ensure that people don't set both ConfigMap and Secret. Currently the webhook validation requires that all Bundles have a configMap set, so this PR would force users to set a configMap target and _optionally_ also a secret.

Reply:
See comment regarding secrets and configmap being mutually exclusive.
I did not intend to change the current behavior of the spec, just doing an additive change that could be later refactored into another version of the API.